threescale_api/resources.py

```diff
@@ -637,6 +672,13 @@ def proxy(self) -> 'Proxies':
     def mapping_rules(self) -> 'MappingRules':
         return MappingRules(parent=self, instance_klass=MappingRule)
 
+    @property
+    def oidc(self):
+        return OIDCConfigs(self)
+
+    @property
+    def backendservices(self):
```

Comment:
I would suggest the name `backend_service`

Reply:
Renamed to backend_usage according name of endpoint.